paper-input-container.html

```diff
@@ -515,8 +515,7 @@
     _handleValue: function(inputElement) {
       var value = this._inputElementValue;
 
-      // type="number" hack needed because this.value is empty until it's valid
```

Comment:
Hmmm, this worries me. If the input has type="number" and a junk input (like `111eee`), then this internally sets the value to the empty string: http://jsbin.com/suruje/edit?html,output

So basically, if you remove this and type the (allowed) string `111eee` into the input, it will say "oh I have no content", and display the label on top of the input.